docs/targets/awseventbridge.md

```diff
@@ -0,0 +1,63 @@
+# Event target for AWS EventBridge
+
+This event target receives arbitrary [CloudEvents][ce] over HTTP and sends them to a [AWS EventBridge partner event
+bus][intro] in a [JSON format][ce-jsonformat].
```

Comment:
to a AWS EventBridge TriggerMesh partner event bus ?

Reply:
The event bus itself is in the hands of the customer, I think including "TriggerMesh" would introduce some confusion, maybe people will assume we control their event bus while we don't?

Reply:
ah right, the bus is theirs, it is the source that is labeled TriggerMesh